projects/storefrontapp-e2e-cypress/cypress/helpers/product-configurator-vc.ts

```diff
@@ -309,17 +317,6 @@ export function clickOnViewInConfigurationAndWait(attribute: string): void {
   cy.wait(GET_CONFIG_ALIAS);
 }
 
-/**
- * Assuming the given attribute is involved in the conflict, it navigates from the conflict group to standard group
- * containing the corresponding attribute and waits for request to finish.
- *
- * @param attribute - Attribute name
- */
-export function clickOnViewInConfigurationAndWait(attribute: string): void {
-  clickOnViewInConfiguration(attribute);
-  cy.wait(GET_CONFIG_ALIAS);
-}
-
```

Comment:
just in case: This method was duplicated (error in vs code). Guess this happened during a manual merge accidentally. hence i removed it here.